src/Components/DisplayArea/DisplayArea.js

```diff
@@ -59,6 +60,7 @@ const DisplayArea = ({ favorites, recipes, handleFavorites }) => {
   return (
```

Comment:
I'm not the biggest fan of ternaries, but you could use one to DRY up lines 41=>54

Reply:
Great call. I made the change.